docs/modern/BabelPluginRelay.md

```diff
@@ -100,3 +100,11 @@ When compiling code for production deployment, the plugin can be configured to i
   ]
 }
 ```
+
+### Alternatives
```

Comment:
If you want to, you could add the babel-macro badge:

[![Babel Macro](https://img.shields.io/badge/babel--macro-%F0%9F%8E%A3-f5da55.svg?style=flat-square)](https://github.com/kentcdodds/babel-macros)

```
[![Babel Macro](https://img.shields.io/badge/babel--macro-%F0%9F%8E%A3-f5da55.svg?style=flat-square)](https://github.com/kentcdodds/babel-macros)
```